### DISCUSSION TITLE
Yazi

### DISCUSSION THREAD

**Discussion by sxyazi** [catppuccin/catppuccin#2189] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

<img width="1117" alt="image" src="https://github.com/catppuccin/catppuccin/assets/17523360/ee82fea6-a9e2-4f96-a2fb-450b7e9ba889">

See https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/theme.toml

Originally, Yazi came with the Catppuccin Macchiato theme, which was beautiful but not very friendly for some light theme users.

I redesigned Yazi's theme system recently, which now defaults to using 16 colors, and it's time to separate Catppuccin from Yazi!

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Yay!

**Comment by sgoudham** [comment]
Closing this as further discussion should continue in #2278
